tests/e2e/interOp/validation_of_operating_modes/bridge_mode/country_code_channel_division/country_code_GB/iOS/test_20mhz.py

```diff
@@ -0,0 +1,1773 @@
+from logging import exception
+import unittest
+import warnings
+from perfecto.test import TestResultFactory
+import pytest
+import sys
+import time
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from appium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+import random
+import string
+import sys
+import allure
+
+if 'perfecto_libs' not in sys.path:
+    sys.path.append(f'../libs/perfecto_libs')
+
+pytestmark = [pytest.mark.interop, pytest.mark.ios, pytest.mark.interop_ios, pytest.mark.interop_country_codeGBI1,
+               pytest.mark.interop_uc_sanity, pytest.mark.bridge]
```

Comment:
Change the marker expressions to the suggestions given below
```suggestion
pytestmark = [pytest.mark.interop, pytest.mark.ios, pytest.mark.interop_ios, pytest.mark.interop_country_code_GB,
                pytest.mark.bridge]
```